api/nodes/serializers.py

```diff
@@ -1773,6 +1776,9 @@ def update_forward_fields(self, obj, validated_data, auth):
         if save_forward:
             forward_addon.save()
 
+            request = self.context['request']
+            forward_addon.owner.check_spam(request.user, {'addons_forward_node_settings__url'}, get_headers_from_request(request))
```

Comment:
I don't really understand this comment, but I don't see the logic in writing migrations into a table that we've agreed is probably going to be depreciated anyway.

Reply:
It appears that the comment was asking for the spam checking to happen inside `forward_addon.save()` (i.e. the model) rather than after save in multiple places.